### PR TITLE
Readme update for running standalone client on Windows

### PR DESCRIPTION
**Pull request by LiFaytheGoblin**
From the issues and from my own experience it seems that the standalone client doesn't work well on windows. I added a section "Windows" to the readme, to point windows users to two possible solutions. 
(1) Cygwin has not worked for me personally, but for @cwillisf it did. 
I tried (2) Wubi(uefi) which worked perfectly well. It is a simple .exe wich installs Ubuntu from within Windows and you can decide for Windows or Ubuntu when booting. No partitioning is needed. In Ubuntu you can still access your files from Windows. You have to install things like npm, node and git again for Ubuntu though. I hope my wording is okay, feel free to change it though. I also wasn't sure about how much information I should provide about the two solutions.

### CODE REVIEW

**Comment by mewtaylor** [comment]
@cwillisf @LiFaytheGoblin fyi: https://github.com/LLK/scratch-www/issues/1332

File: README.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ make deploy
 ### Current issues with the development
 We're currently in the process of transitioning into this web client from Scratch's existing structure. As we transition, there are going to be some issues along the way that relate to how this client needs to interact with the existing infrastructure to work properly in production.
 
+#### FALLBACK ####
```
Comment (liam4) [inline README.md]: Humble nitpick, other headings in the README don't have the following `####`, i.e.:

```markdown
<!-- Bad -->
#### FALLBACK ####
#### Windows ####

<!-- Good -->
#### FALLBACK
#### Windows
```

Comment (mewtaylor) [inline README.md]: @liam4 nice catch, though I would not say this is a matter of "bad" or "good", just convention. @LiFaytheGoblin would you mind removing the trailing `####`? Probably best to follow the other header conventions for now.

Comment (liam4) [inline README.md]: Yep, I had thought to go with `OLD` and `NEW`, but that didn't seem right, either. `NORMAL` and `YOU` seemed to be putting too much blame or guilt.. writing code comments that you're actually presenting isn't easy :)

**Comment by St19Galla** [comment]
Happy first pull request @LiFaytheGoblin 🎉🎉🎉🎉🎉

File: README.md (hunk 1 above)
Comment (mewtaylor) [inline README.md]: @liam4 gotcha – I think another way would be to say `Currently in the PR` and `Repo convention`, perhaps :).

**Comment by mewtaylor** [comment]
Haha you beat me to it @St19Galla :).

File: README.md (hunk 1 above)
Comment (LiFaytheGoblin) [inline README.md]: Thanks for the hint! I can remove these extra ### :)

**Pull request merged**